### DISCUSSION TITLE
Best way to make changes

### DISCUSSION THREAD

**Discussion by severn-leon** [lucky-media/landtamic#31] [Q&A] [answered]
What is the best way to modify the starter kit?  For example I want to change some of the markup in the Hero Set, should I just change hero_section_4.antlers.html?  Is there the possibility that this will get over-written by an update?  Is there some kind of over-ride system that preserves the original code and prevents changes getting over-written by updates?

**Comment by lokmanm** [comment, marked as answer]
Hey @severn-leon,

Starter Kits are meant to be a `starting point`, meaning after installation you are free to make any changes as you wish.

To update a Starter Kit you will have to monitor the releases and make the necessary code changes because unlike add-ons you can't update them through composer.  

You can read a bit more here on docs regarding this topic:
https://statamic.dev/starter-kits#statamic-starter-kits-vs-wordpress-themes
